src/app/components/dashboard/dashboard-content/dashboard-content.component.scss

```diff
@@ -3,16 +3,4 @@
   background: -webkit-linear-gradient(#f5ac28, #e07e7e);
   -webkit-background-clip: text;
   -webkit-text-fill-color: transparent;
-  font-size: 26px;
 }
-
-.btn {
-  font-size: 1.2rem;
-  padding-top:.5%;
-  height: 45px;
-}
-
-.text-dark-black {
-  font-size: 3rem;
-}
```

Comment:
Why these changes in this PR @Kajol-Kumari

Reply:
As due to issue with my browser's view, i had made these changes wrong in the dashboard. It's for reverting those.

Reply:
I see. Try to make separate PR for different issues but as you have added here now, lets merge with these changes too.